src/test/java/Pages/AllSongsPage.java

```diff
@@ -0,0 +1,126 @@
+package Pages;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

For example 

![image](https://github.com/Jamesxj4/A53-QA-Automation-Framework/assets/104761482/e4b12bb9-f042-457a-9df7-771f0da4681f)